src/list-action-group/list-action-group.component.html

```diff
@@ -0,0 +1,11 @@
+<div>
+  <button type="button" class="editor-btn-delete" (click)="onDelete.emit()">
```

Comment:
What function do you mean exactly? because logic of deleting things in not inside of this component, so this is just a button group and notifies parent when a button clicked.